main.go

```diff
@@ -114,6 +116,7 @@ func main() {
 	}
 	eng.Logger.Info().Msgf("clientset successfully created in %s", time.Since(start))
 
+	eng.Logger.Info().Msgf("starting 'Watcher' and 'Suspender' routines")
```

Comment:
I'm not sure this logline is useful as the first thing `watcher` and `suspender` do is log they started

Reply:
Sure, but what if they don't?

By example, in the issue https://github.com/govirtuo/kube-ns-suspender/pull/53#issuecomment-1079519601, I tried some "fix" on my own and I ended up with: 

![image](https://user-images.githubusercontent.com/1590399/160282316-64ea941c-bce0-4d48-b7fb-b6548b7d0f25.png)

And you can see that the `watcher` did not start as expected and since there was a `Mutex` issue, nothing else happened.

From my experience, even it if sounds duplicated, it is better to log before and after an action.